src/renderers/shaders/ShaderChunk.js

```diff
@@ -106,6 +106,7 @@ import uv2_vertex from './ShaderChunk/uv2_vertex.glsl.js';
 import worldpos_vertex from './ShaderChunk/worldpos_vertex.glsl.js';
 
 import * as background from './ShaderLib/background.glsl.js';
+import * as backgroundCube from './ShaderLib/backgroundCube.glsl.js';
```

Comment:
Rather that hacking into the existing background shader, I created a new background cube shader that supports PMREM for blurriness.

Reply:
I'm okay with this approach. Maybe it makes it easier to maintain the background related shader code.

Reply:
Yes, that is the hope. :-)